2023-risks/m1-insecure-authentication-authorization

```diff
@@ -0,0 +1,124 @@
+---
+
+layout: col-sidebar
+title: "M1: Insecure Authentication/Authorization"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Threat agents that exploit authentication and authorization vulnerabilities typically do so through automated attacks that use available or custom-built tools.
```

Comment:
I would add something on attackers' possible motivation, because without, they would not be a threat.